.script/tests/KqlvalidationsTests/CustomTables/GitLabAccess.json

```diff
@@ -0,0 +1,48 @@
+{
+  "Name": "GitLabAccess",
+  "Properties": [
+    {
+      "Name": "TimeGenerated",
+      "Type": "DateTime"
+    },
+    {
+      "Name": "EventTime",
+      "Type": "DateTime"
+    },
+    {
+      "Name": "IPAddress",
+      "Type": "String"
+    },
+    {
+      "Name": "RequestVerb",
+      "Type": "String"
+    },
+    {
+      "Name": "URI",
+      "Type": "String"
+    },
+            {
+      "Name": "HTTPVersion",
+      "Type": "String"
+    },
+    {
+      "Name": "HTTPReferer",
+      "Type": "String"
+    },
+        {
+      "Name": "ResponseCode",
+      "Type": "String"
+    },
+    {
+      "Name": "BytesSent",
+      "Type": "String"
+    },
+        {
+      "Name": "EntityType",
+      "Type": "String"
+    },  
+    {
+      "Name": "UserAgent",
+      "Type": "String"
+    },
+  ]
```

Comment:
end bracket '}' is missing, hence the validation error, please fix it.

Reply:
done! good catch, thanks!